src/components/products/Products.component.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { useEffect } from 'react';
+
+import productsFromApi from '@/assets/fake_data/products.json';
+import { ProductsListComponent } from '@/components/productList/ProductsList.component.tsx';
+import type { ProductProps } from '@/interfaces/ProductProps.ts';
+
+export const ProductsComponent = () => {
+    const page = 0;
```

Comment:
I suggest removing it, cause it could be the reason for content jumping.
Here is screen cast with my thoughts https://drive.google.com/file/d/1MSeF50Uj_4NH9oZOVXT-k-xCqLv-gnyg/view?usp=sharing

Reply:
You were right, that was the reason, thanks for the advice. Fixed